### PR TITLE
Yk/update docker images version

### PR DESCRIPTION
**Pull request by YKharouni**
- Update helm charts and app versions of ArmoniK
- update mangosh image version

fix : #876

### CODE REVIEW

File: infrastructure/modules/aws/eks/cluster-autoscaler.tf [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  namespace = "kube-system"
-  chart     = "cluster-autoscaler"
-  #repository = "https://kubernetes.github.io/autoscaler"
-  repository = "${path.module}/charts"
-  version    = "9.14.0"
+  name       = "armonik"
+  namespace  = "kube-system"
+  chart      = "cluster-autoscaler"
+  repository = "https://kubernetes.github.io/autoscaler"
+  #repository = "${path.module}/charts"
```
Comment (lzianekhodja) [inline infrastructure/modules/aws/eks/cluster-autoscaler.tf:7]: Delete this comment

File: infrastructure/modules/aws/eks/instance_refresh.tf [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-  namespace = "kube-system"
-  chart     = "aws-node-termination-handler"
-  #repository       = "https://aws.github.io/eks-charts"
-  repository       = "${path.module}/charts"
-  version          = "0.17.0"
+  name       = "aws-node-termination-handler"
+  namespace  = "kube-system"
+  chart      = "aws-node-termination-handler"
+  repository = "https://aws.github.io/eks-charts"
+  #repository       = "${path.module}/charts"
```
Comment (lzianekhodja) [inline infrastructure/modules/aws/eks/instance_refresh.tf:7]: Delete this comment

File: infrastructure/modules/monitoring/metrics-server/metrics-server.tf [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-  namespace = var.namespace
-  chart     = "metrics-server"
-  #repository = "https://kubernetes-sigs.github.io/metrics-server/"
-  repository = "${path.module}/charts"
-  version    = "3.8.2"
+  name       = "metrics-server"
+  namespace  = var.namespace
+  chart      = "metrics-server"
+  repository = "https://kubernetes-sigs.github.io/metrics-server/"
+  #repository = "${path.module}/charts"
```
Comment (lzianekhodja) [inline infrastructure/modules/monitoring/metrics-server/metrics-server.tf:6]: Delete this comment

File: infrastructure/modules/persistent-volumes/efs/efs-csi.tf [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
-  namespace = kubernetes_service_account.efs_csi_driver.metadata.0.namespace
-  chart     = "aws-efs-csi-driver"
-  #repository = "https://kubernetes-sigs.github.io/aws-efs-csi-driver/"
-  repository = "${path.module}/charts"
-  version    = "2.3.0"
+  name       = "efs-csi"
+  namespace  = kubernetes_service_account.efs_csi_driver.metadata.0.namespace
+  chart      = "aws-efs-csi-driver"
+  repository = "https://kubernetes-sigs.github.io/aws-efs-csi-driver/"
+  #repository = "${path.module}/charts"
```
Comment (lzianekhodja) [inline infrastructure/modules/persistent-volumes/efs/efs-csi.tf:6]: Delete this comment

File: infrastructure/quick-deploy/aws/all/armonik.tf [hunk 5]
```diff
@@ -62,6 +62,7 @@ module "armonik" {
     kubernetes_secret.shared_storage,
     kubernetes_secret.deployed_object_storage,
     kubernetes_secret.deployed_table_storage,
-    kubernetes_secret.deployed_queue_storage
+    kubernetes_secret.deployed_queue_storage,
+    module.mongodb
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/aws/all/armonik.tf:66]: Delete the explicit dependency to MongoDB

File: infrastructure/quick-deploy/aws/armonik/parameters.tfvars [hunk 6]
```diff
@@ -201,7 +201,7 @@ ingress = {
 authentication = {
   name                    = "job-authentication-in-database"
   image                   = "125796369274.dkr.ecr.eu-west-3.amazonaws.com/mongosh"
-  tag                     = "1.5.4"
+  tag                     = "1.23.3"
```
Comment (lzianekhodja) [inline infrastructure/quick-deploy/aws/armonik/parameters.tfvars:204]: The versionof mongosh is 1.7.1

**Review by lzianekhodja — COMMENTED** [review]
Retest

**Review by lzianekhodja — APPROVED** [review]

**Comment by lemaitre-aneo** [comment]
For the terraform issue, you might want to use the action to install the dependencies. I've just created a PR to fix the issue: https://github.com/aneoconsulting/Armonik.Action.Deploy/pull/2

**Review by lzianekhodja — APPROVED** [review]

**Review by lzianekhodja — APPROVED** [review]

**Pull request merged**